cluster-operator/src/main/java/io/strimzi/operator/cluster/model/metrics/StrimziReporterMetricsModel.java

```diff
@@ -18,6 +18,10 @@
  * Represents a model for components with configurable metrics using Strimzi Reporter
  */
 public class StrimziReporterMetricsModel {
+    /**
+     * Fully Qualified Class Name (FQCN) of the Strimzi Kafka Prometheus Metrics Reporter.
+     */
+    public static final String METRICS_REPORTER_FQCN = "io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter";
```

Comment:
Following `CrusieControlMetricsReporter` convention, I would call it `KAFKA_PROMETHEUS_METRICS_REPORTER`.

Reply:
Rerfactored